### ISSUE TITLE
Problem with ConBee Firmware Update

### ISSUE THREAD

**Issue by Nastras** [dresden-elektronik/deconz-rest-plugin#417]
Hello,
i have the following firmware update issue with my ConBee stick.

It will be a continuous firmware update in the GUI offers, as soon as I make the firmware update, the stick is no longer reachable in deconz.

There is no way to reconnect it with deconz, only when I backup my SD card is the stick reconnected.

I also tried over the terminal to make an update and get this answer:

`pi@HomeKitServer:/usr/bin $ sudo GCFFlasher_internal -l
 ftdi_sio
GCFFlasher V2_11 (c) dresden elektronik ingenieurtechnik gmbh 2017/12/10

       1  FTDI device found

 device | vendor | product | serial  | description
--------|--------|---------|---------|----------------------
failed to open FTDI device (0) status = 3

pi@HomeKitServer:/usr/bin $ sudo GCFFlasher_internal -f
 ftdi_sio
GCFFlasher V2_11 (c) dresden elektronik ingenieurtechnik gmbh 2017/12/10
usage: GCFFlasher <options>
options:
 -r              force device reset without programming
 -f <firmware>   flash firmware file
 -d <device>     device to use
 -sn <serial>    FTDI device with serial number to use
 -l              list devices
 -h -?           print this help
done`

What can I do to fix the problem?

**Comment by manup** [comment]
Here are a few things to check?

1) Which version of deCONZ are you running?
2) How do you start deCONZ &mdash; start menu, systemd deconz-gui script or commandline?
3) Please provide the output of: 

```sh
    $ ls -l /dev/ttyUSB0

    # ... and

    $ groups
```

**Comment by Nastras** [comment]
Thank you for your prompt reply,

here is the data you wanted.

![b4dfc84f-9848-4c7a-b930-0a91c9d2561b](https://user-images.githubusercontent.com/27686185/36277752-15b164dc-1292-11e8-9922-a888a848441d.jpeg)
![b6ab5db8-b677-4616-8d36-a140a9b6a8bf](https://user-images.githubusercontent.com/27686185/36277754-15d5fc52-1292-11e8-8958-8f2e2941c8c8.jpeg)

I hope that's enough for a first analysis?

Da ich in einem anderen Post gesehen habe das du Deutsch sprichst, können wir gerne auch Deutsch schreiben wenn du möchtest?

Gruß Nastra

**Comment by Nastras** [comment]
Start is with autostart is it the gui script?

Here is a picture of the update we offered.
![8f91e1bf-70f7-404e-9d1d-b373a88bd063](https://user-images.githubusercontent.com/27686185/36278138-26f21574-1293-11e8-81dd-03cce244ab25.jpeg)

**Comment by manup** [comment]
Lets keep it english since the audience here is mostly international :)

I'm a bit confused about the screenshots and operating system, is this a Raspbian ARM or Debian x86?

**Comment by Nastras** [comment]
Ok 👍

Its Debian x86

**Comment by manup** [comment]
Debian x86 is not officialy supported, albeit similar to Ubuntu there might be problems.

I've just tried to update ConBee from 0x26190500 to 0x261e0500 on Ubuntu directly from deCONZ and it worked. However the start script has some Ubuntu specific parts like check dependencies and setup the user to the dialout group, which we can change to *Debian like* parts to support both, but in your case this doesn't seem to be the problem. 

To Update manually:

    ... close deCONZ

    `$ sudo GCFFlasher_internal -d 0 -f /usr/share/deCONZ/firmware/deCONZ_Rpi_0x261e0500.bin.GCF`

**Comment by Nastras** [comment]
Hello manup,

I was not aware that the debain desktop is not officially supported but it works fine now. Thanks for the hint!

I noticed two things that are probably a bug?

After a reboot the ConBee Stick does not connect by itself. Only after I removed the hardware by hand from the computer and added again at the USB port.
In addition, the connect button works in deconz although the ConBee is displayed.

Unfortunately, I still have a problem creating a user on the Homebridge-hue plugin.

I have described the problem here:

https://github.com/ebaauw/homebridge-hue/issues/289

I asked Erik, but maybe you have an idea too?
